sdk/eventhub/event-hubs/src/eventData.ts

```diff
@@ -229,6 +229,9 @@ export function toRheaMessage(data: EventData, partitionKey?: string): RheaMessa
 export interface EventData {
   /**
    * @property The message body that needs to be sent.
+   * If the application reading the events is not using this SDK,
+   * consider setting the `body` to a `Buffer` for better cross-language
```

Comment:
"setting body to Buffer" makes it sound like "Buffer" is something that I can directly set the body to.
How about "consider converting your body payload to bytes or Buffer and then setting this property to it"?

Also, why "consider"? Why not simply say that it is the right thing to do?

Reply:
Good points, I updated the text and got rid of 'consider'.